extensions/strategies/trust_distrust/strategy.js

```diff
@@ -0,0 +1,135 @@
+var z = require('zero-fill')
+  , n = require('numbro')
+
+module.exports = function container (get, set, clear) {
+  return {
+    name: 'trust_distrust',
+    description: 'Sell when price higher than $sell_min% and highest point - $sell_threshold% is reached. Buy when lowest price point + $buy_threshold% reached.',
+
+    getOptions: function () {
+      this.option('period', 'period length', String, '30m')
```

Comment:
I feel like this `min_periods` shouldn't be part of the strategy, as it's needed in trade.js... but I added it, as I see that all other strategies, except for SAR, have also added it (but don't actually do anything with it).

Good catch!